config/locales/en.yml

```diff
@@ -743,6 +743,16 @@ en:
       some_not_crypto_html: >-
         Note that some software does not need to use cryptographic
         mechanisms.
+        If your project produces software that (1) includes,
+        activates, or enables encryption functionality, and
+        (2) might be released from the United States (US) to outside
+        the US or to a non-US-citizen, you may be legally required
+        to take a few extra steps.
+        Typically this just involves sending an email.
+        For more information, see the encryption section of
+        <a href="https://www.linuxfoundation.org/wp-content/uploads/2020/07/UnderstandingOpenSourceTechnologyandUSExportControls_Whitepaper_070220.pdf"
```

Comment:
This URL doesn't work for me...?

Reply:
Thanks for letting me know!!

It seems to have moved to:
https://www.linuxfoundation.org/wp-content/uploads/UnderstandingOpenSourceTechnologyandUSExportControls_Whitepaper_070220-1.pdf

I was trying to direct link to the report, but it looks like that isn't a stable URL. So I think we should just link here instead:
https://www.linuxfoundation.org/resources/publications/understanding-us-export-controls-with-os-projects/

Reply:
Thanks so much for the report. I've tweaked the pull request to a URL that I believe is stable.